test_fixture/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .browser import ChromeWrapper, FirefoxWrapper, NodeWrapper, SeleniumWrapper
```

Comment:
Maybe let's call this pyodide_pytest or pyodode-test-runner (if we want to be able to use this without pytest, not sure) even if it's not yet a proper pytest plugin?

test_fixture makes me think of a folder where tests for fixtures are rather than a folder that defines fixtures.